recipes/rapidcheck/all/conanfile.py

```diff
@@ -123,6 +123,7 @@ def _module_file_rel_path(self):
                             "conan-official-{}-targets.cmake".format(self.name))
 
     def package_info(self):
+        self.cpp_info.set_property("cmake_file_name", "rapidcheck")
```

Comment:
```suggestion
```
This is not needed

Reply:
nop, but it's explicit, at least for me it's easier to maintain recipes explicitly declaring official config files.